style/themes/lcars-picard.css

```diff
@@ -0,0 +1,5882 @@
+  padding: var(--gap50) var(--gap50);
+  flex-shrink: 0;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > li.user-header > p {
+  color: var(--color-red-alert);
+  font-size: 26px;
+  margin-top: 0;
+  text-transform: uppercase;
+  text-align: left;
+  margin-bottom: 0;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > li.user-header > p::after {
+  content: "A black hole for Internet advertisements";
+  display: block;
+  font-size: 1.5rem;
+  color: var(--color-quinternary);
+}
+
+/* --- footer */
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer {
+  display: flex;
+  flex-direction: column;
+  padding: var(--gap) var(--gap) calc(var(--gap) * 2) var(--gap);
+  background: var(--color-background);
+  border: var(--border-width) solid var(--color-secondary);
+  border-top: 0;
+  border-left: 0;
+  border-right-width: var(--border-panel-width);
+  border-bottom-right-radius: 0;
+  margin-top: var(--border-width);
+  position: relative;
+}
+
+@media (min-width: 576px) {
+  .navbar-nav > .user-menu > .dropdown-menu > .user-footer {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer hr {
+  width: 100%;
+}
+
+@media (min-width: 576px) {
+  .navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link {
+    width: calc(50% - var(--gap50));
+    margin-block: var(--gap50);
+  }
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link:first-child {
+  width: 100%;
+  background-color: var(--color-success-dark) !important;
+  color: var(--color-success);
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link:first-child:hover {
+  background-color: var(--color-link) !important;
+  color: #fff;
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link:last-child {
+  width: 100%;
+  background-color: var(--color-supplement-01) !important;
+  color: var(--color-danger-bright);
+}
+
+.navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link:last-child:hover {
+  background-color: var(--color-danger) !important;
+  color: #fff;
+}
+
+@media (max-width: 768px) {
+  .navbar-nav > .user-menu > .dropdown-menu > .user-footer .btn-link + .btn-link {
+    margin-top: var(--gap50);
+  }
+}
+
+/* --- fake corners */
+@media (min-width: 768px) {
+  .navbar-nav > .user-menu > .dropdown-menu > li.user-header::after,
+  .navbar-nav > .user-menu > .dropdown-menu > .user-footer::after {
+    display: block;
+    content: "";
+    background-color: var(--color-background);
+    width: var(--gap);
+    border-radius: 0 var(--radius50) 0 0;
+    position: absolute;
+    inset: 0 calc(var(--gap50) * -1) 0 auto;
+    z-index: 5;
+  }
+
+  .navbar-nav > .user-menu > .dropdown-menu > .user-footer::after {
+    border-radius: 0 0 var(--radius50) 0;
+  }
+}
+
+/* --- additional visual tweaks */
+.navbar-custom-menu > .navbar-nav > li > .dropdown-menu::after {
+  content: "LCARS theme by @MichalSvatos";
```

Comment:
@MichalSvatos - can you remove this please? No issue with the Author attribution in the header of the file.

The Pull Request / commits / release notes will provide credit

Reply:
Oh, sure! I've already removed one in the footer, I forgot about this one.